lib/demos/note_demos_view.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_full_learn/101/image_learn.dart';
+
+class NoteDemos extends StatelessWidget {
+  const NoteDemos({super.key});
+  final _title = "Create your first note";
+  final _description = "Add a note";
+  final _createNote = "Create a Note";
+  final _importNotes = "İmport Notes";
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color.fromARGB(255, 201, 237, 255),
+      appBar: AppBar(
+          // systemOverlayStyle: SystemUiOverlayStyle.dark,
```

Comment:
Can you please commented code since its not in use?

Reply:
Done.